examples/java/testcontainer/src/test/java/com/grafana/example/TestcontainerTest.java

```diff
@@ -29,10 +34,10 @@ void setUp() {
   }
 
   @Test
-  void testExportMetric() {
-    // Howto:
-    // 1. The test with a really long timeout
-    // 2. Go to the Grafana UI
+  void testExportMetricsAndTraces() throws IOException, InterruptedException {
+    // How to debug:
+    // 1. Run the test with a really long timeout (update the awaitility argument)
+    // 2. Go to the Grafana UI (login with admin:admin)
```

Comment:
login is disabled by default

Reply:
yea you are right, this was a side effect of me not having the latest container locally, thanks!